website/src/components/ElevatorPitch/index.tsx

```diff
@@ -0,0 +1,103 @@
+"use client";
+
+import Link from "next/link";
+import Image from "next/image";
+import { ReactNode, useState } from "react";
+import { manrope } from "@/lib/fonts";
+import type { CustomFlowbiteTheme } from "flowbite-react";
+import {
+  HiMiniShieldCheck,
+  HiMiniPresentationChartLine,
+} from "react-icons/hi2";
+import { HiLightningBolt, HiGlobe } from "react-icons/hi";
+
+export default function ElevatorPitch() {
+  const [selectedOption, setSelectedOption] = useState(5);
+
+  const data = [
+    {
+      title: "Fast and secure.",
+      desc: "Built on WireGuard® to be 3-4 times faster than OpenVPN.",
+      icon: <HiLightningBolt className="min-w-8 h-8 text-primary-400" />,
+    },
+    {
+      title: "Scales with your business.",
+      desc: "Deploy two or more gateways for automatic load balancing and failover.",
+      icon: (
+        <HiMiniPresentationChartLine className="min-w-8 h-8 text-primary-400" />
+      ),
+    },
+    {
+      title: "Zero attack surface.",
+      desc: "Firezone's holepunching tech establishes tunnels on-the-fly at the time of access.",

```

Comment:
The description here needs to be tweaked to support the `Minimal attack surface` claim. What do you think of this?

```suggestion
      desc: "No need to open firewall ports. Firezone hole-punches connections directly between your users and resources.",

```